app/views/articles/_sidebar.html.erb

```diff
@@ -46,7 +46,7 @@
           <a href="/terms">Terms of Use</a>
           <a href="/contact">Contact</a>
           <a href="/faq">FAQ</a>
-          <a href="/code-of-conduct">Code of Conduct</a>
+          <a href="/code-of-conduct" data-cy-href="codeofconduct">Code of Conduct</a>
```

Comment:
`data-cy` should be adequate enough for e2e tests. It is the recommended data attribute to use by the Cypress team.